backend/src/main/java/org/cryptomator/hub/license/LicenseHolder.java

```diff
@@ -69,7 +75,13 @@ public DecodedJWT get() {
 	public boolean isExpired() {
 		return Optional.ofNullable(license) //
 				.map(l -> l.getExpiresAt().toInstant().isBefore(Instant.now())) //
-				.orElse(CommunityLicenseConstants.IS_EXPIRED);
+				.orElseGet(() -> {
+					if (!managedInstance) {
+						return CommunityLicenseConstants.IS_EXPIRED;
+					} else {
+						return ManagedInstanceNoLicenseConstants.IS_EXPIRED;
+					}
+				});
```

Comment:
I wonder if this is really necessary? If there is no license, it's always "not expired", right? Can't we just say `.orElse(false)`?

Reply:
Right, firstly Managed wasn't expired and after changing, I missed to change this block back, thx!

Reply:
Fixed in https://github.com/cryptomator/hub/pull/174/commits/9a7e1d46f452775cccec5351b8291a600f66f10e